docs/content/dev/getting-started.md

```diff
@@ -1,60 +1,138 @@
 # Getting started
 
-This document will help you with setting up the development environment for HedgeDoc 2.
-
-## Preparation
-
-### Preparing the directory
+To run Hedgedoc 2.0 you need three components: the backend, the frontend and the reverse proxy.
+
+Backend and Frontend are included in the [HegdeDoc repo](https://github.com/hedgedoc/hedgedoc). The reverse proxy can be chosen by preference. For development, we
+recommend caddy and the provided configuration.
+
+## Quick guide for development setup
+
+This describes the easiest way to start a local development environment. For other deployments follow the description
+below.
+To run Hedgedoc 2.0 you need three components: the backend, the frontend and the reverse proxy.
+
+Backend and Frontend are included in the [HegdeDoc repo](https://github.com/hedgedoc/hedgedoc). The reverse proxy can be chosen by preference. For development, we
+recommend caddy and the provided configuration.
+
+## Quick guide for development setup
+
+This describes the easiest way to start a local development environment. For other deployments follow the description
+below.
+
+1. Install Node.js (at least Node 14, we recommend Node 18) and [Yarn](https://yarnpkg.com/getting-started/install)
+2. Install Caddy (select one of the two options)
+   - [Download](https://caddyserver.com/) and place the `caddy` binary in `hedgedoc/dev-reverse-proxy`. Ensure it is executable with `chmod +x caddy`
+   - Install Caddy with you package manager
+3. Clone [our repository](https://github.com/hedgedoc/hedgedoc.git) and go into its directory
+     ```shell
+     git clone https://github.com/hedgedoc/hedgedoc.git
+     cd hedgedoc
+     ```
+4. Install the dependencies in repo root directory with `yarn install`
+5. Go to `hedgedoc/commons` directory with `cd ../commons`
+6. Build the commons package with `yarn build`
+7. Goto `hedgedoc/backend` directory with `cd ../backend`
+8. Create the `.env` config file by copying the example: `cp .env.example .env`
+9. Add a value to `HD_SESSION_SECRET` in the .env file. This can be any string, which has to be a secure password for production but can be set to simple string for debug purpose.
+10. Add the following lines
```

Comment:
Add? To what?
Either you change the code below so the user gets instructed to add these lines to the `.env` file or you change "add" to "execute".